conan_patches/onnxruntime/conandata.yml

```diff
@@ -0,0 +1,13 @@
+patches:
+  "1.18.1":
+    - patch_file: "patches/android-64.diff"
+      patch_description: |
+        Fixes error for arm64-v8a targets:
+        onnxruntime/core/mlas/lib/mlasi.h:366:11: error: unknown type name 'bfloat16_t'
+    - patch_file: "patches/msvc-x86.diff"
+      patch_description: |
+        Fixes detection of x86 windows targets during cross-compilation.
```

Comment:
does it make sense to upstream this patch? Or upstream dropped 32-bit support?

Reply:
I would first wait to see if the models run on windows-x86 targets - I have no suitable hardware to test (honestly, I was surprised it even compiled. Hopefully it will run!)